webapp/app/elster_client/elster_errors.py

```diff
@@ -137,3 +140,21 @@ class EricaIsMissingFieldError(GeneralEricaError):
     def __init__(self):
         self.message = _l('form.lotse.input_invalid.MissingFieldsInputValidationError')
 
+
+class EricaRequestTimeoutError(InternalServerError):
+    """Exception raised when a Timeout reached in Erica"""
+
+    def __init__(self, original_exception=None):
+        self.message = _l('erica.timeout.error')
+        self.original_exception = original_exception
+        super(EricaRequestTimeoutError, self).__init__()
+
+
+class EricaRequestConnectionError(InternalServerError):
```

Comment:
Why did you decide to inherit from InternalServerError and not GeneralEricaError? These could also inherit from one common exception (e.g. EricaRequestError or sth). Then you would only need error handler in routes instead of two

Reply:
Because of thinking it should be a 500. I could inherit from RequestException instead.

Reply:
Yes, makes sense that they inherit from InternalServerError (or even RequestException) and not GeneralEricaError.

In my head we should still have a way to differentiate these two from other RequestExceptions with an error handler in routes.py